package/network/services/dnsmasq/files/dnsmasq.init

```diff
@@ -604,6 +631,7 @@ dnsmasq_start()
 
 	config_get_bool ADD_LOCAL_DOMAIN "$cfg" add_local_domain 1
 	config_get_bool ADD_LOCAL_HOSTNAME "$cfg" add_local_hostname 1
+	config_get_bool ADD_LOCAL_GATENAME "$cfg" add_local_gatename 0
```

Comment:
Yes `add_wan_hostname` would be parallel. For 90%, **add_** is most meaningful, because ISP often dont bother with IP4 `c64-100-57-7.city.state.example.net` and never IP6 due to the wide complexity.